cmd/root.go

```diff
@@ -52,6 +52,9 @@ any client SSL certificates.`,
 
 // runSignalWrapper watches for SIGTERM and SIGINT and interupts execution if necessary.
 func runSignalWrapper(cmd *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		return errInvalidInvocation
+	}
```

Comment:
So we can leave this for now, but I think we'll probably end up yanking it out at some point when things like "fuse" or even config files are added since args will no longer be required.